.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,42 @@
+name: Auto merge
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+
+jobs:
+    auto-merge:
+        name: Auto-Merge
+
+        runs-on: ubuntu-latest
+
+        needs:
+            - test
```

Comment:
This does not work, it only knows about jobs in the same workflow

Reply:
Didn't know this, I rollback the split.